cli/nodeutil.ts

```diff
@@ -17,6 +17,11 @@ export var targetDir: string = process.cwd();
 //When running the Electron app, this will be based on the initial value
 export var pxtCoreDir: string = path.join(targetDir, "node_modules/pxt-core")
 
+export function setTargetDir(dir: string) {
+    targetDir = dir;
+    pxtCoreDir = path.join(targetDir, "node_modules/pxt-core");
```

Comment:
yes please to logging.
this area of code is giving me headaches last night trying to find out what was breaking electron